openapi/openapi.yaml

```
@@ -1670,6 +1670,57 @@ paths:
         "500":
           description: On server side errors.
 
+  /graph/sync-job/{docuement_type}:
```

```suggestion
  /graph/sync-job:
```
I think this is not needed anymore.